glu/glu.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "glu",
```

Comment:
this should be the shared modules

Reply:
Resolved in 852ee02abdf6790e8ea52769975fbd208d1ae847.